packages/0x.js/src/types.ts

```diff
@@ -175,6 +175,7 @@ export interface OrderStateWatcherConfig {
     eventPollingIntervalMs?: number;
     expirationMarginMs?: number;
     cleanupJobIntervalMs?: number;
+    stateLayer: BlockParamLiteral;
```

Comment:
Add description for `stateLayer` in the above comment. Let's mention that it defaults to `pending`.

Reply:
Oh, it should also be `optional`.